teams/alumni.toml

```diff
@@ -29,6 +29,7 @@ members = [
     "jseyfried",
     "levex",
     "niconii",
+    "nrc",
```

Comment:
You should be able to add yourself to the alumni of the core team too

Reply:
I didn't know that's a thing, looks like I should have read the great big comment at the top of the file :-)